.gitignore

```diff
@@ -33,3 +33,4 @@ Changelog.html
 doc/*.1
 doc/man-*.html
 doxygen-warnings.log
+venv_cpython/
```

Comment:
The lack of trailing new line (not a problem), just curious, is that something that the commit hook doesn't do? Because I think it should.

Reply:
I think it doesn't

Reply:
I prepared an update on factory, which makes it handle .gitignore files too.